content/docs/csidriver/features/powerscale.md

```diff
@@ -192,6 +192,8 @@ spec:
       storage: 5Gi
 ```
 
+>Starting from CSI PowerScale driver version 2.2, it is allowed to create PersistentVolumeClaim from VolumeSnapshot with different isi paths i.e., isi paths of the new volume and the VolumeSnapshot can be different.
```

Comment:
isi paths, should we need to camel case , I can see the reference as isiPath everywhere

Reply:
no, it is a reference to the path on isilon and not the actual attribute names(the name is different in SC and VSC)